code-generators/java-types/plain-pojo/src/test/resources/serialised_model.json

```diff
@@ -0,0 +1,62 @@
+{
+  "aud" : "https://passport.core.stubs.account.gov.uk",
+  "exp" : null,
+  "iat" : null,
+  "iss" : "https://review-p.build.account.gov.uk",
+  "jti" : null,
+  "nbf" : 1690816091,
+  "sub" : "urn:fdc:gov.uk:2022:954bc117-731b-41cd-86cf-dfb4e7940fce",
+  "vc" : {
+    "context" : null,
```

Comment:
Note that in the plain POJO this property name is incorrect - it should be `@context` (not `context`). It doesn't really matter since afaict we never actually populate this property - it's a W3 VC property https://www.w3.org/TR/vc-data-model/#contexts

Reply:
Good spot! This is resolved in the test now. The serialisation wasn't configured properly, but now the serialised JSON is the same between the annotated and plain modules.